pkg/skaffold/parser/config.go

```diff
@@ -231,7 +235,9 @@ func processEachConfig(ctx context.Context, config *latest.SkaffoldConfig, cfgOp
 				}
 			}
 		}
-		newOpts := configOpts{file: cfgOpts.file, profiles: depProfiles, isRequired: required, isDependency: cfgOpts.isDependency}
+		isRemoteDependency := d.GitRepo != nil || d.GoogleCloudStorage != nil
```

Comment:
Not sure but, is this needed? I think we are always using the value from the flag in line 282 (isRemoteCfg), and we are overwriting the value from here(?)

Reply:
You're right, removed. Thanks!